frontend/src/api/common/logEvent.ts

```diff
@@ -0,0 +1,28 @@
+import axios from 'api';
+import { ErrorResponseHandler } from 'api/ErrorResponseHandler';
+import { AxiosError } from 'axios';
+import { ErrorResponse, SuccessResponse } from 'types/api';
+import { EventSuccessPayloadProps } from 'types/api/events/types';
+
+const logEvent = async (
+	eventName: string,
+	attributes: Record<string, unknown>,
+): Promise<SuccessResponse<EventSuccessPayloadProps> | ErrorResponse> => {
+	try {
+		const response = await axios.post('/event', {
+			eventName,
+			attributes,
```

Comment:
we should update the useAnalytics hook to send all the data through this end-point  now ?

Reply:
This would be a simple API call. We would require a hook for this.